.travis.yml

```
@@ -2,7 +2,7 @@ language: java
 scala:
 - 2.12.4
 jdk:
-- oraclejdk8
+- oraclejdk9
```

```suggestion
- openjdk11
```
? or however we get to OpenJDK 11